packages/fiori-elements-writer/src/data/manifestSettings.ts

```diff
@@ -34,6 +36,17 @@ export function extendManifestJson<T>(
         ].includes(feApp.template.type)
     ) {
         templatePath = TemplateType.ListReportObjectPage;
+
+        // starting with UI5 v1.94.0, contextPath should be used instead of 'entitySet' in manifest for v4 LROP based apps
```

Comment:
Is it clear (from the context) that the new logic is restricted to V4?

Reply:
Please see: https://github.com/SAP/open-ux-tools/pull/1339/files/3a8d5f93153ab00f6158734ce939da9ec1c441eb#diff-7b7cfaa4b7ae65f0bebe254a5487d1cff0effaa3a8ee544f0bf93d83ac9d4e88R30

Reply:
OK